integrations/apify.md

```diff
@@ -0,0 +1,244 @@
+---
+layout: integration
+name: Apify
+description: Extract data from the web and automate web tasks using Apify-Haystack integration.
+authors:
+  - name: apify
+    socials:
+      github: apify
+      twitter: apify
+      linkedin: apify
+pypi: https://pypi.org/project/apify-haystack
+repo: https://github.com/apify/apify-haystack
+type: Data Ingestion
+report_issue:  https://github.com/apify/apify-haystack/issues
+logo: /logos/apify.png
+version: Haystack 2.0
+toc: true
+---
+
+### Table of Contents
+
+- [Overview](#overview)
+- [Installation](#installation)
+- [Usage](#usage)
+    - [ApifyDatasetFromActorCall on its own](#apifydatasetfromactorcall-on-its-own)
+    - [ApifyDatasetFromActorCall in a RAG pipeline](#apifydatasetfromactorcall-in-a-rag-pipeline)
+- [License](#license)
+
+## Overview
+
+[Apify](https://apify.com) is a web scraping and data extraction platform. 
+It helps automate web tasks and extract content from e-commerce websites, social media (Facebook, Instagram, TikTok), search engines, online maps, and more. 
+Apify provides more than two thousand ready-made cloud solutions called Actors.
+
+## Installation
+
+Install the Apify-haystack integration:
+```bash
+pip install apify-haystack
+```
+
+## Usage
+
+Once installed, you will have access to more than two thousand ready-made apps called Actors at [Apify Store](https://apify.com/store)
+
+- Load a dataset from Apify and convert it to a Haystack Document
+- Extract data from Facebook/Instagram and save it in the InMemoryDocumentStore
+- Crawl websites, scrape text content, and store it in the InMemoryDocumentStore
+- Retrieval-Augmented Generation (RAG): Extracting text from a website & question answering
+
+The integration implements the following components:
+- `ApifyDatasetLoader`: Load a dataset created by an Apify Actor
+- `ApifyDatasetFromActorCall`: Call an Apify Actor, load the dataset, and convert it to Haystack Documents
+- `ApifyDatasetFromTaskCall`: Call an Apify task, load the dataset, and convert it to Haystack Documents
+
+You need to have an Apify account and an Apify API token to run this example.
+You can start with a free account at [Apify](https://apify.com/) and get your [Apify API token](https://docs.apify.com/platform/integrations/api#api-token).
+
+In the examples below, specify `apify_api_token` and run the script.
+
+
+### ApifyDatasetFromActorCall on its own
+
+
+Use Apify's [Website Content Crawler](https://apify.com/apify/website-content-crawler) to crawl a website, scrape text content, and convert it to Haystack Documents.
```

Comment:
Feel free to mention Apify Store here with a sentence like this: 
```suggestion
Use Apify's [Website Content Crawler](https://apify.com/apify/website-content-crawler) to crawl a website, scrape text content, and convert it to Haystack Documents. You can browse other Actors in [Apify Store](https://apify.com/store)
```

Reply:
Excellent! Thank you